packages/workbox-webpack-plugin/index.js

```diff
@@ -62,6 +62,25 @@ class WorkboxBuildWebpackPlugin {
       config.globPatterns = ['**/*.{html,js,css}'];
     }
 
+    if (compilation.options.output.publicPath && !config.manifestTransforms) {
+      const publicPath = compilation.options.output.publicPath;
+      const compiledAssets = [];
+      for (let key in compilation.assets) {
+        if (compilation.assets.hasOwnProperty(key)) {
+          compiledAssets.push(path.resolve(publicPath, './' + key));
```

Comment:
Regarding how publicPath should be treated, my understanding based on https://webpack.js.org/configuration/output/#output-publicpath is that the value is treated as a string prefix, not as a value that should be joined with `url.resolve()` or `path.join()`. If the value ends in `/` as recommended, then this will be equivalent, but if the value doesn't end in `/` then you'll end up interpreting `publicPath` different in the precache manifest than how it's being interpreted by Webpack.

@goldhand was this a consideration when you wrote `sw-precache-webpack-plugin` (which uses `url.resolve()`)?